src/cclib/parser/logfileparser.py

```diff
@@ -86,20 +86,23 @@ def __init__(self, source, pos=0):
             self.src.seek(0, 2)
             self.size = self.src.tell()
             self.src.seek(pos, 0)
-            self.pos = pos
 
-        except:
+        except (AttributeError, IOError):
```

Comment:
Comment above the start of the try block sounds like a test case that throws an ```UnsupportedOperation``` should be found.